### ISSUE TITLE
Unexpected token: const getOutput = async (page, request)

### ISSUE THREAD

**Issue by solarsilk** [spatie/browsershot#188]
on my server,  I'm using node v9.9.0 (npm v5.6.0)
```
$nodePath = '/home/split19/nodejs/bin/node';
$npmPath = '/home/split19/nodejs/bin/npm';
Browsershot::html($html1)
  ->setNodeBinary($nodePath)
  ->setNpmBinary($npmPath)
  ->fullPage()
  ->hideBackground()
  ->deviceScaleFactor(3)
  ->border(40, '00000000', Manipulations::BORDER_EXPAND)
  ->save($p1img);
```
getting error:
```
The command "PATH=$PATH:/usr/local/bin NODE_PATH=`/home/split19/nodejs/bin/node /home/split19/nodejs/bin/npm root -g` /home/split19/nodejs/bin/node '/home/split19/my.perfectplanbook.com/vendor/spatie/browsershot/src/../bin/browser.js' '{"url":"file:\/\/\/tmp\/0243469001524168604\/index.html","action":"screenshot","options":{"type":"png","path":"\/home\/split19\/my.perfectplanbook.com\/storage\/page1.png","args":[],"viewport":{"width":800,"height":600,"deviceScaleFactor":3},"displayHeaderFooter":false,"fullPage":true,"omitBackground":true}}'" failed.\n ◀

Exit Code: 1(General error)

Working directory: /home/split19/my.perfectplanbook.com/public\n

Output:\n
================


Error Output:\n
================
/home/split19/my.perfectplanbook.com/vendor/spatie/browsershot/bin/browser.js:5\n
const getOutput = async (page, request) => {\n
                        ^

SyntaxError: Unexpected token (\n
    at exports.runInThisContext (vm.js:53:16)\n
    at Module._compile (module.js:373:25)\n
    at Object.Module._extensions..js (module.js:416:10)\n
    at Module.load (module.js:343:32)\n
    at Function.Module._load (module.js:300:12)\n
    at Function.Module.runMain (module.js:441:10)\n
    at startup (node.js:139:18)\n
    at node.js:968:3\n
"""
```

thanks for any help. greatly appreciated!

**Comment by SudoGetBeer** [comment]
What is the output of:
/home/split19/nodejs/bin/node -v

It seems your node version is too old.

**Comment by SudoGetBeer** [comment]
You can set the node and npm path via the setNodeBinary($nodePath) and setNpmBinary($npmPath) methods.

**Comment by solarsilk** [comment]
Thanks @SudoGetBeer.

I used `nvm which 9.9.0` to get the path to v9.9.0 which I didn't realize was installed in .nvm folder (my inexperience, but it makes perfect sense): `/home/split19/.nvm/versions/node/v9.9.0/bin/node` 

I changed the code to:
```
                    ->setNodeBinary('/home/split19/.nvm/versions/node/v9.9.0/bin/node')
                    ->setNpmBinary('/home/split19/.nvm/versions/node/v9.9.0/bin/npm')
```
This got rid of the error. Thank you for the help!

And thanks @freekmurze for the excellent package and all your amazing open source packages at Spatie! Postcard is in the mail.
